diff --git a/i18n_menu/i18n_menu.module b/i18n_menu/i18n_menu.module
index 8f91fc5..1c381c3 100644
--- a/i18n_menu/i18n_menu.module
+++ b/i18n_menu/i18n_menu.module
@@ -719,15 +719,17 @@ function i18n_menu_form_menu_edit_item_alter(&$form, &$form_state) {
  * Add a "translate" link in operations column for each menu item.
  */
 function i18n_menu_form_menu_overview_form_alter(&$form, &$form_state) {
-  foreach (element_children($form) as $element) {
-    if (substr($element, 0, 5) == 'mlid:') {
-      $mlid = $form[$element]['#item']['mlid'];
-      if (i18n_get_object('menu', $mlid)->get_translate_access()) {
-        $form[$element]['operations']['translate'] = array(
-          '#type' => 'link',
-          '#title' => t('translate'),
-          '#href' => "admin/structure/menu/item/{$mlid}/translate",
-        );
+  if (i18n_menu_mode($form['#menu']['menu_name'], I18N_MODE_MULTIPLE)) {
+    foreach (element_children($form) as $element) {
+      if (substr($element, 0, 5) == 'mlid:') {
+        $mlid = $form[$element]['#item']['mlid'];
+        if (i18n_get_object('menu', $mlid)->get_translate_access()) {
+          $form[$element]['operations']['translate'] = array(
+            '#type' => 'link',
+            '#title' => t('translate'),
+            '#href' => "admin/structure/menu/item/{$mlid}/translate",
+          );
+        }
       }
     }
   }
